bindgen/codegen/mod.rs

```diff
@@ -1137,8 +1137,10 @@ impl<'a> CodeGenerator for Vtable<'a> {
                         _ => panic!("Function signature type mismatch"),
                     };
 
-                    // FIXME: Is there a canonical name without the class prepended?
-                    let function_name = function_item.canonical_name(ctx);
+                    let mut function_name = function_item.canonical_name(ctx);
+                    if ctx.options().disable_name_namespacing {
+                        function_name = function_name.rsplit_once('_').unwrap().1.to_string();
```

Comment:
I don't think you can rely on fixing it up here. What if you get something like this?

```cpp
class Test_Class {
public:
	virtual void virtual_method() = 0;
};
```

You probably have to change the `canonical_name` impl